js/hackathon/HoneySpam/honeyspam.js

```diff
@@ -24,8 +24,8 @@
  */
 
 document.observe('dom:loaded', function () {
-    if (typeof $('url') !== 'undefined' && $('url') !== null) {
-        $('url').hide();
+    var mhhsInput = $$('.mhhs-input');
+    if (mhhsInput.length > 0 && typeof mhhsInput !== 'undefined' && mhhsInput !== null) {
```

Comment:
You should change the order and put `mhhsInput.length > 0` at the end. Imho this only works, because $$ returns always an array.

Or just remove the other tests.